edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -294,6 +295,21 @@ static string GetLocalConfigFilePath(IConfiguration configuration, ILogger logge
             return localConfigPath;
         }
 
+        static Option<ulong> GetStorageMaxTotalWalSizeIfExists(IConfiguration configuration)
+        {
+            ulong storageMaxTotalWalSize = 0;
+            try
+            {
+                storageMaxTotalWalSize = configuration.GetValue<ulong>(Constants.StorageMaxTotalWalSize);
```

Comment:
There's no way to test for the presence of the config value without having to catch an exception? Like a `TryGetValue`?

Reply:
I doesn't have trygetvalue, this only throws when the value can not be converted to ulong. If the value is not set in the env variable it returns 0. For other settings edgeHub initialization fails it the value can't be converted to what is expected, so I can remove the try catch and have the same behavior